_gettingstarted/python-hunter.md

```diff
@@ -18,20 +18,20 @@ author:
 ## Introduction
 Python Hunter is a versatile code tracing toolkit developed for debugging, logging, inspection, It is not meant for measuring the  coverage. It offers a simple Python API, a comfortable terminal API, and a command-line tool for attaching to processes. If you have any bugs in your code, and if you worry how to trace them, this is the best package that can help you. It will help you in tracing and understanding the flow of your code.Debugging the code is very challenging especially for beginners, it will behave as a debugging assistant that can help you in reavealing the hidden paths of your program that happens during execution.
 
- **Installation**
+###Installation
```

Comment:
- There needs to be a space after the last # 
- Also use ## for headings and ### for subheadings. 

Example - 
![image](https://github.com/tankala/learn-pip-trends/assets/67141217/c6faaa8d-d787-48b0-8a7b-edf006c5608a)
![image](https://github.com/tankala/learn-pip-trends/assets/67141217/72654d9f-48a5-47f3-95e8-d1590f884a16)

Headings starting with ## will only be displayed on the sidebar, subheadings(###) will not be displayed